documentation/modules/exploit/windows/local/ms16_014_wmi_recv_notif.md

```diff
@@ -0,0 +1,36 @@
+## Intro
+
+This module exploits a uninitialized stack variable vulnerability present in the WMI subsystem  of `ntoskrnl`.  The vulnerability is described by MS16-014 and CVE-2016-0040.  The module allows the user to elevate privileges (NT AUTHORITY\SYSTEM)
```

Comment:
without looking at the code, does this only work against win7 (due to offsets etc) or is it only TESTED against win7.  The MS bulletin showed vista and 2008 and many others vulnerable.

Reason I ask, is that this intro is a good place to add something like:
This module has been tested against the following systems:
- Windows 7 x64 sp1
- Foobarbaz

This module should work, but has not been tested against:
- Windows 2008


That way a user will know what it definitely works for, and may (to make their own risk assessment).